prediction_market_agent/agents/crewai_subsequential_agent/crewai_agent_subquestions.py

```diff
@@ -0,0 +1,152 @@
+import typing as t
+
+from crewai import Agent, Task, Process, Crew
+from crewai_tools import SerperDevTool
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket
+from pydantic import BaseModel
+
+from prediction_market_agent.agents.abstract import AbstractAgent
+from prediction_market_agent.agents.crewai_subsequential_agent.prompts import *
+
+search_tool = SerperDevTool()
+
+
+class Outcomes(BaseModel):
+    outcomes: list[str]
+
+
+class ProbabilityOutput(BaseModel):
+    decision: str
+    p_yes: float
+    p_no: float
+    confidence: float
+
+
+class CrewAIAgentSubquestions(AbstractAgent):
+    def __init__(self) -> None:
+        self.researcher = Agent(
```

Comment:
I see gpt 3.5 turbo defined on multiple places in the code, but it doesn't like it's being passed to crewai agents. What version are they actually using? From the OpenAI dashboard it seems like it's got 4 turbo:

![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/29b11a56-1cd1-46e2-8ed0-149c39329cef)

Reply:
crewai reads gpt-model from ENV. Fixed now.